src/families/algorand/Rewards/index.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { View, StyleSheet } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+
+import type { Account } from "@ledgerhq/live-common/lib/types";
+
+import {
+  getAccountUnit,
+  getAccountCurrency,
+} from "@ledgerhq/live-common/lib/account";
+import AccountSectionLabel from "../../../components/AccountSectionLabel";
+import Info from "../../../icons/Info";
+import colors from "../../../colors";
+import LText from "../../../components/LText";
+import Button from "../../../components/Button";
+import CurrencyUnitValue from "../../../components/CurrencyUnitValue";
+import CounterValue from "../../../components/CounterValue";
+import { ScreenName, NavigatorName } from "../../../const";
+
+type Props = {
+  account: Account,
+};
+
+const RewardsSection = ({ account }: Props) => {
+  const { rewards } = account.algorandResources || {};
```

Comment:
this shouldn't be needed. do an invariant to not have bad flowtype inference (rewards must be a weird type here?)